code/game/objects/structures/crates_lockers/largecrate_vr.dm

```diff
@@ -35,4 +35,41 @@
 							 "<span class='notice'>You hear splitting wood.</span>")
 		qdel(src)
 	else
-		return attack_hand(user)
\ No newline at end of file
+		return attack_hand(user)
+
+/obj/structure/largecrate/animal/pred
+	name = "Predator carrier"
+	held_type = /mob/living/simple_animal/catgirl
+
+/obj/structure/largecrate/animal/pred/New() //This is nessesary to get a random one each time.
+
+	held_type = pick(/mob/living/simple_animal/hostile/bee,
+						/mob/living/simple_animal/catgirl,
+						/mob/living/simple_animal/catgirl,
+						/mob/living/simple_animal/catgirl,
+						/mob/living/simple_animal/hostile/frog,
+						/mob/living/simple_animal/horse,
+						/mob/living/simple_animal/hostile/panther,
+						/mob/living/simple_animal/hostile/snake,
+						/mob/living/simple_animal/hostile/wolf,
+						/mob/living/simple_animal/hostile/bear,
+						/mob/living/simple_animal/hostile/bear/brown,
+						/mob/living/simple_animal/hostile/carp,
```

Comment:
Pike is on the other list, and the large white carp is no no, since its a boss monster